### PR TITLE
feat(oracle):add cli to query latest price of token

### PR DESCRIPTION
**Pull request by leonz789**
## Description

Add cli to query latest price of a token
----

Closes #XXX

## Summary by CodeRabbit

- **New Features**
  - Introduced a command to query the latest price of a specific token.
  
- **Enhancements**
  - Added backend support for querying the latest price of a token, including request and response structures.

### CODE REVIEW

**Review by mikebraver — APPROVED** [review]
LGTM

File: x/oracle/client/cli/query_prices.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ import (
 func CmdShowPrices() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "show-prices [token-id]",
-		Short: "shows a prices",
+		Short: "shows all prices of a specific token",
```
Comment (MaxMustermann2) [inline x/oracle/client/cli/query_prices.go]: Maybe say that it's for the latest round?

**Review by MaxMustermann2 — APPROVED** [review]

File: x/oracle/client/cli/query_prices.go (hunk 1 above)
Comment (leonz789) [reply x/oracle/client/cli/query_prices.go]: udpated

**Comment by cloud8little** [comment]
Test passed:

`exocored q oracle show-latest-price 1`
```
price:
  decimal: 18
  price: "291597000000"
  round_id: "13"
  timestamp: "2024-07-08 07:12:31"
```

**Review by cloud8little — APPROVED** [review]

**Pull request merged**